Lib/test/test_sys.py

```diff
@@ -1882,10 +1890,11 @@ def test_pythontypes(self):
         # symtable entry
         # XXX
         # sys.flags
-        # FIXME: The +3 is for the 'gil', 'thread_inherit_context' and
-        # 'context_aware_warnings' flags and will not be necessary once
-        # gh-122575 is fixed
-        check(sys.flags, vsize('') + self.P + self.P * (3 + len(sys.flags)))
+        # FIXME: The +3 is for the 'gil', 'thread_inherit_context',
```

Comment:
TODO: misworded.  change "The +3 is for" to "The non_sequence_fields adjustment is for the:" and list each field on its own comment line.